docker-compose.yml

```diff
@@ -23,4 +25,5 @@ services:
       - OCTOBOX_DATABASE_USER=postgres
       - OCTOBOX_DATABASE_PASSWORD=development
       - OCTOBOX_DATABASE_HOST=database.service.octobox.internal
+      - MEMCACHEDCLOUD_SERVERS=memcached.service.octobox.internal
```

Comment:
If we want to mirror a "production" environment in development, we may also need to set the username and password for the memcached service ([file](https://github.com/octobox/octobox/blob/master/config/environments/production.rb#L56-L59))